src/main/resources/application.properties

```diff
@@ -1,15 +1,22 @@
 server.port=8080
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/patternpedia
```

Comment:
is this a change that needs to be reflected in the [docker-compose file](https://github.com/PatternAtlas/pattern-atlas-docker)?

Reply:
An ENV variable which sets the database name is introduced to the dockerfile [here](https://github.com/PatternAtlas/pattern-atlas-api/blob/9d99a3a6ec89e32d2bd66aa7da3123f2cd9ed412/Dockerfile#L16). The default name is still `postgres` but can be changed via the docker-compose file.